examples/email-auth-local-storage/README.md

```diff
@@ -65,3 +67,30 @@ $ pnpm run dev
 ```
 
 This command will run a NextJS app on port 3000. If you navigate to http://localhost:3000 in your browser, you can follow the prompts to start an email auth.
+
+### 4/ Stackblitz Example
+
+You will need to create a Stackblitz account to run the following example
+
+Example Link: https://stackblitz.com/edit/stackblitz-starters-pyyw59
+
+#### Prerequisites
+
+To use the example you need the following
+
+- Turnkey Organization
+- API key for the root user (both public and private key needed)
+- Turnkey Suborg with an email user
+
+#### Set Env Variables
+
+Set the following environment variables in the `.env.local` file
+
+- `API_PUBLIC_KEY` --> Set this to be the public key of the API key for the root user for the org you've created
+- `API_PRIVATE_KEY` --> Set this to be the private key of the API key for the root user for the org you've created
+- `NEXT_PUBLIC_ORGANIZATION_ID` --> Set this to be the Org ID of the org you've created
+- `NEXT_PUBLIC_BASE_URL` --> This will be preset with https://api.turnkey.com
+
+#### Directions to use
```

Comment:
realistically, I think users may jump directly to the stackblitz and follow the README instructions there, so as long as we include these instructions in the stackblitz itself, we should be golden

Reply:
we can probably remove this portion of the stackblitz's README: 
<img width="287" alt="image" src="https://github.com/user-attachments/assets/a3c2d691-d9e5-47d6-aad1-d024f2f42ea7">

Reply:
Yeah I'll make sure to edit the stackblitz readme's!

Reply:
Readme's updated